diff -ur gal-0.19.2.orig/gal/e-table/Makefile.am gal-0.19.2/gal/e-table/Makefile.am
--- gal-0.19.2.orig/gal/e-table/Makefile.am	Sat May 18 08:51:24 2002
+++ gal-0.19.2/gal/e-table/Makefile.am	Sat May 18 08:53:28 2002
@@ -155,4 +155,4 @@
 EXTRA_DIST =			\
 	 $(icons)		\
 	 $(glade_DATA)
-#	 sample.table		\
+#	 sample.table		
Only in gal-0.19.2/gal/e-table: Makefile.am~
diff -ur gal-0.19.2.orig/tests/Makefile.am gal-0.19.2/tests/Makefile.am
--- gal-0.19.2.orig/tests/Makefile.am	Tue Oct  2 16:19:39 2001
+++ gal-0.19.2/tests/Makefile.am	Sat May 18 08:54:05 2002
@@ -1,5 +1,4 @@
 
-
 INCLUDES =					\
 	-I$(top_srcdir)                         \
 	-I$(top_srcdir)/gal                     \
@@ -21,37 +20,12 @@
 noinst_PROGRAMS = \
 	test-table-1 test-tree-1 $(vfs_programs) test-tree-3 test-shortcut-bar test-define-views test-unicode
 
-test_table_1_LDADD =	$(LDADD)
-test_table_1_LDFLAGS = $(LDFLAGS)
-
-test_tree_1_LDFLAGS = $(LDFLAGS)
-test_tree_1_LDADD = $(LDADD)
-
-test_shortcut_bar_LDFLAGS = $(LDFLAGS)
-test_shortcut_bar_LDADD = $(LDADD)
-
-if GAL_HAVE_VFS
-#test_tree_2_LDFLAGS = $(LDFLAGS)
-#test_tree_2_LDADD = $(LDADD)		\
-#	$(GNOME_VFS_LIBS)
-endif
-
-test_tree_3_LDFLAGS = $(LDFLAGS)
-test_tree_3_LDADD = $(LDADD) 
-
 test_shortcut_bar_SOURCES = 	\
 	test-shortcut-bar.c
 
 test_define_views_SOURCES = 	\
 	test-define-views.c
 
-test_define_views_LDFLAGS = $(LDFLAGS)
-test_define_views_LDADD   = $(LDADD)
-
-test_unicode_LDFLAGS = $(LDFLAGS)
-test_unicode_LDADD   = $(LDADD)
-
-
 
 
 
