diff -urN gal-2.1.14.orig/configure.in gal-2.1.14/configure.in
--- gal-2.1.14.orig/configure.in	2004-08-15 11:32:01.852433448 +0200
+++ gal-2.1.14/configure.in	2004-08-15 11:32:34.847417448 +0200
@@ -67,7 +67,7 @@
 fi
 AC_SUBST(GAL_HAVE_ALLOCA_H)
 
-ALL_LINGUAS="am az be bn ca cs da de el en_CA en_GB es et eu fi fr ga gl gu hr hu it ja ko lt lv mn ms nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta tr uk vi zh_CN zh_TW" 
+ALL_LINGUAS="am az be bn ca cs da de el en_CA en_GB es et eu fi fr ga gl gu hr hu it ja ko lt lv mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta tr uk vi zh_CN zh_TW" 
 
 GETTEXT_PACKAGE=gal-${GAL_API_VERSION}
 AC_SUBST(GETTEXT_PACKAGE)
